### DISCUSSION TITLE
Polyline Hitbox Question

### DISCUSSION THREAD

**Discussion by btilford** [fleaflet/flutter_map#1982] [Q&A] [answered]
How is the hitbox calculated? It seems like the actual rendered line isn't included but maybe the border lines are? Maybe just the edges or the line?

**Comment by josxha** [comment, marked as answer]
- The width of the polyline inclusive the border gets calculated.
- The max distance to the center of the line gets calculated (half the polyline width with border, minimum the `minimumHitbox` property).
- Every segment of the line gets checked if the distance is in the maximum distance threshold, if yes the line has been clicked.
- If no segment is nearby, the line has not been clicked.

https://github.com/fleaflet/flutter_map/blob/master/lib/src/layer/polyline_layer/painter.dart#L50-L62
